linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/ResponseClassifiers.scala

```diff
@@ -39,7 +37,8 @@ class NonRetryable5XXInitializer extends ResponseClassifierInitializer {
 object NonRetryable5XXInitializer extends NonRetryable5XXInitializer
 
 class AllSuccessfulConfig extends ResponseClassifierConfig {
-  def mk: ResponseClassifier = ClassifiedRetries.Default
+  def mk: ResponseClassifier = ??? //ClassifiedRetries.Default
```

Comment:
`ClassifiedRetries.Default` is of type `http.ResponseClassifier`, not `h2.ResponseClassifier`. This could make `ResponseClassifiers.Default` which should have that behaviour?